src/globus_cli/commands/group/set_policies.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.termio import formatted_print
+
+from ._common import group_id_arg
+
+
+@click.option(
+    "--high-assurance/--not-high-assurance",
+    help="Flag if the group will enforce high assurance access policies or not.",
```

Comment:
I think the behavior of this flag is that it is binary, `True/False`.
We actually want ternary, `True/False/None` (so that nothing is modified if it is not set). I'm hoping we have an example around of doing this -- or that I've forgotten and the `click` automatically gives us the `True/False/None` behavior.

Reply:
Ah right, I need to add a default=None, thank you!